__tests__/app.test.js

```diff
@@ -229,20 +229,69 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
-describe('GET /api/articles/:article_id/comemnts', () => {
+
+describe('GET /api/articles/:article_id/comments', () => {
     test('200: should return an array of comments for the given article_id with the correct properties', () => {
         return request(app)
         .get("/api/articles/1/comments")
         .expect(200)
         .then(({body})=> {
             const commentsArr = body.articleComments
-            articleComments.forEach((comments) => {
+            expect(commentsArr).toHaveLength(11)
+            commentsArr.forEach((comments) => {
                 expect(comments).toEqual(expect.objectContaining({
-                    comment_id: 1,
+                    comment_id: expect.any(Number),
                     votes: expect.any(Number),
                     created_at: expect.any(String), author: expect.any(String),
                     body: expect.any(String)
+                }))
+            })
+        })
+    });
+    test('404: valid but non-existent article should return no article found', () => {
+        return request(app)
+        .get("/api/articles/60/comments")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toEqual('no article found for this ID')
+        })
+    });
+    test('404: should return "route not found" for incorrect path', () => {
+        return request(app)
+        .get("/api/articles/60/commentssss")
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe("route not found")
+        })
+    });
+    test('200: Should return an empty array for a valid article that exists but has no comments attached to it', () => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articleComments).toEqual([])
+        })
+    });
+    test('400: invalid article_id returns bad request', () => {
+        return request(app)
+        .get('/api/articles/dragons/comments')
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe("bad request")
+        })
+    });
+});
 
+describe('GET /api/articles - comment count feature', () => {
+    test('200: the articles array should now have a comment count for each article', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            const allArticles = body.articles
+            allArticles.forEach((article) => {
+                expect(article).toEqual(expect.objectContaining({
+                    comment_count: expect.any(String)
```

Comment:
But you also need to prove that the comment count has been calculated correctly for at least one of the articles, not just that it is a string